apps/core/scenes/bookingDetail/tripDetails/Header.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, StyleSheet } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+
+import type { Header as BookingType } from './__generated__/Header.graphql';
+
+type Props = {|
+  +data: ?BookingType,
+|};
+
+const Header = (props: Props) => {
+  const bookingId = props.data?.bookingId ?? '';
+  return (
+    <View style={[styles.row, styles.container]}>
+      <View style={styles.row}>
+        <View style={styles.circle} />
+        <Text type="success" size="small">
+          {props.data?.status}
+        </Text>
+      </View>
+      <Text type="secondary" size="small">
+        {`#${bookingId}`}
+      </Text>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    justifyContent: 'space-between',
+  },
+  circle: {
```

Comment:
Should we create a component for that?

Reply:
I don't think we will need it anywhere else, but if we ever do, we can extract it.